scipy/optimize/_linprog.py

```diff
@@ -606,6 +606,7 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
             _check_result(sol['x'], sol['fun'], sol['status'], sol['slack'],
                           sol['con'], lp.bounds, tol, sol['message']))
         sol['success'] = sol['status'] == 0
+        sol['sensitivity'] = OptimizeResult(sol['sensitivity'])
```

Comment:
Originally I said that it should be a subclass of `OptimizeResult` because I thought `OptimizeResult` itself had some required fields, but it looks like it doesn't. The documentation states that [OptimizeResult](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.OptimizeResult.html) will have a bunch of fields that `linprog` already doesn't return, so I think we just need to adjust the documentation at some point to reflect the fact that `OptimizeResult` isn't guaranteed to have anything in particular.

Reply:
Whoops -- forgot to subclass!  Not sure if subclassing will do anything useful here if the docs for `OptimizeResult` should be changed anyway